legend/metrics_library/metrics/mysql_56_rds_metrics.j2

```diff
@@ -170,13 +178,17 @@ panels:
     description: The percentage of Burst Balance remaining
     targets:
       {% for dimension in data %}
+        {% if dimension.get('is_disk_gp2') is none or dimension.get('is_disk_gp2') is sameas true %}
```

Comment:
Would this work as well? `dimension.get('is_disk_gp2', True)`

Reply:
jinja2 does not allow default values in get. 
Also, dimension.get('is_disk_gp2') is none is added for backward compatibility.

Reply:
Can we set a default in schema?

Reply:
done, added dimension.get('is_disk_gp2', true)

It is working as expected